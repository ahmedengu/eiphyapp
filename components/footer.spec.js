import { mount } from 'enzyme';
import Footer from './footer';

function setup(props = {}) {
  return mount(<Footer {...props} />);
}

describe('Footer Tests', () => {
  it('should work', () => {
    const actual = setup();
    expect(actual).toBeTruthy();
  });

  it('should match snapshot', () => {
    const actual = setup();
    expect(actual.html()).toMatchSnapshot();
  });
});
