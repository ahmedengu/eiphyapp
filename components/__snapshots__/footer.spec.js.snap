// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Tests should match snapshot 1`] = `
<footer class="mastfoot mt-auto">
  <div class="inner">
  </div>
</footer>
`;
